upgrading.md

```diff
@@ -17,7 +17,15 @@ command from the root of your app (the same directory that contains the
 $ flutter upgrade
```

Comment:
`flutter upgrade` updates both channel and packages as before, per your recommendation.